### DISCUSSION TITLE
Expiring session

### DISCUSSION THREAD

**Discussion by digitalbuugle** [supabase/supabase#1982] [Questions] [answered]
Hello I need help? 
I'm a flutter developer I recently begin to use Supabase for my project it was cool and all until I came across this recover session thing which is really annoying and Hard to deal with I authenticated my user but it just keeps me out every hour and so please help me this is really huge problem for me

**Comment by silentworks** [comment, marked as answer]
Can you try upgrading to the latest version of the supabase library as from what I remember this issue was fixed some short time ago. If it's still there after upgrading report back here, please.

It's useful to state part of the issue you are having the the title of the post instead of just "I need help".

**Reply by inian** [reply]
Hey @buugle, please confirm if upgrading to the latest library fixes it. A reproducible example would help us a lot too. Mark this as answered till we have more information.
